karma.conf.js

```diff
@@ -20,6 +20,6 @@ module.exports = function(config) {
     colors: true,
     logLevel: config.LOG_WARN,
     singleRun: true,
-    browsers: ['Chrome'],
+    browsers: ['ChromeHeadless'],
```

Comment:
What is this change for ?

Reply:
The integration test tool (Karma) will launch a browser (Chrome) to run the test.
It works in our local machine and CircleCI environment as they are desktop environments.
I changed Chrome to ChromeHeadless to lower the requirements of the test environment because it can event work in non-desktop environments (CLI-only, without the window system)